lib/tinybucket/resource/hooks.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+module Tinybucket
+  module Resource
+    class Hooks < Tinybucket::Resource::Base
+      def initialize(repo, options)
+        @repo = repo
+        @args = [options]
+      end
+
+      # Find the hook
+      #
+      # @param branch [String]
+      # @param options [Hash]
+      # @return [Tinybucket::Model::Hook]
+      def find(branch, options = {})
```

Comment:
According [this code](https://github.com/hirakiuc/tinybucket/pull/103/files#diff-69700f138c358e5f100d265f0bbd80caR39),  I think that `branch` argument should be renamed to `hook`. 🤔